pom.xml

```diff
@@ -97,18 +97,16 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <spring.platformVersion>4.3.24.RELEASE</spring.platformVersion>
+        <spring.platformVersion>4.3.25.RELEASE</spring.platformVersion>
         <ognl.version>3.1.23</ognl.version>
         <asm.version>7.1</asm.version>
         <tiles.version>3.0.8</tiles.version>
         <tiles-request.version>1.0.7</tiles-request.version>
-        <log4j2.version>2.11.2</log4j2.version>
-        <jackson.version>2.9.9</jackson.version>
-        <!-- FIXME: drop once a new 2.9.10 will be out and all packages will be in sync -->
-        <jackson-databind.version>2.9.9.3</jackson-databind.version>
+        <log4j2.version>2.12.1</log4j2.version>
+        <jackson.version>2.9.10</jackson.version>  <!-- 2.9.10 contains the 2.9.9.x jackson-databind fixes -->
```

Comment:
IMO the extra comment at EOL should better be part of the commit comment b/c it has no direct relevance for future changes. Instead it explains the current change/commit.

Reply:
Yeah, such comment is not relevant anymore

Reply:
Hi.

I thought it might make sense to remove it when the next Jackson update happened (since it isn't always clear that x.y includes everything that x.y.z has).  Either way it isn't a big deal since details are in the commit comments anyway.

Latest commit removes that comment, let me know if there's anything else out-of-place.